activerecord/bin/test

```diff
@@ -1,20 +1,11 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-COMPONENT_ROOT = File.expand_path("..", __dir__)
-require_relative "../../tools/test"
-
-module Minitest
-  def self.plugin_active_record_options(opts, options)
-    opts.separator ""
-    opts.separator "Active Record options:"
-    opts.on("-a", "--adapter [ADAPTER]",
-            "Run tests using a specific adapter (sqlite3, sqlite3_mem, mysql2, postgresql)") do |adapter|
```

Comment:
If try to explain it with code, we need to override `TestUnit::Runner.attach_before_load_options`. However, I think that do not have to do that for internal tools. 
Instead, I think it is a good to add explain to the document(maybe `RUNNING_UNIT_TESTS.rdoc` is appropriate), what about it?